### DISCUSSION TITLE
(Probably) Incompatible with KDE

### DISCUSSION THREAD

**Discussion by Jing-Linux** [davatorium/rofi#1474] [Q&A]
I installed Garuda Linux KDE, and installed awesome. Then I go to https://github.com/manilarome/the-glorious-dotfiles to borrow the awesomewm config he made. I uses surreal config. As in screenshot, the rofi menu should be full screen, but it isn't the case with me. It is rendered at half of the screen, and the icons are small and with a white background. If a screenshot is needed I will try my best to get it. Any help is appreciated.

**Comment by DaveDavenport** [comment]
What rofi version, it could be incompatibility.

**Reply by DaveDavenport** [reply]
also if you run awesome, how is it kde incompatibility?
I am very confused.
